AutoGr/settings-sample.py

```diff
@@ -1,211 +0,0 @@
-"""
```

Comment:
You should `git pull` first to get the latest files. Then, after making changes, make sure you do not add with `git add .` but by adding individual changed files separately.

Reply:
This file shouldn't be affected at all in your commit here.

Reply:
Next time, I'll be careful

Reply:
No problem. 

Since there is too much mess here, I suggest you create a new branch off of master and make changes there. (You can copy paste code from here for whatever is needed.) Rest is up to you how you do it =]